scripts/globals/magianobjectives.lua

```diff
@@ -7,647 +7,299 @@
-        reqs = { mobid = set{ 17457236 } }
-    },
-
-[ 779] = {  -- Gloomanita
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17137821 } }
-    },
-
-[ 780] = {  -- Mischievous Micholas
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17281149 } }
-    },
-
-[ 781] = {  -- Cactuar Cantautor
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17289560 } }
-    },
-
-[ 782] = {  -- Erebus
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17334553, 17338599 } }
-    },
-
-[ 783] = {  -- Skuld
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17178924 } }
-    },
-
-[1646] = {  -- Chesma
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17195485 } }
-    },
-
-[ 941] = {  -- Be'Hya Hundredwall
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17363258 } }
-    },
-
-[ 942] = {  -- Jolly Green
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17092898 } }
-    },
-
-[ 943] = {  -- Trembler Tabitha
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17588278 } }
-    },
-
-[ 944] = {  -- Seww the Squidlimbed
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17498301 } }
-    },
-
-[ 945] = {  -- Ankabut
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17137705 } }
-    },
-
-[ 946] = {  -- Okyupete
-        check = checks.mobkill,
-        reqs = { mobid = set{ 16879847 } }
-    },
-
-[ 947] = {  -- Urd
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17178923 } }
-    },
-
-[ 948] = {  -- Lamprey Lord
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17138078, 17146178 } }
-    },
-
-[1788] = {  -- Chesma
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17195485 } }
-    },
-
-[ 891] = {  -- Desmodont
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17571870 } }
-    },
-
-[ 892] = {  -- Moo Ouzi the Swiftblade
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17395816 } }
-    },
-
-[ 893] = {  -- Ni'Zho Bladebender
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17223797 } }
-    },
-
-[ 894] = {  -- Bugbear Strongman
-        check = checks.mobkill,
-        reqs = { mobid = set{ 16822423, 16822427 } }
-    },
-
-[ 895] = {  -- La Velue
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17121576 } }
-    },
-
-[ 896] = {  -- Hovering Hotpot
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17596628 } }
-    },
-
-[ 897] = {  -- Yacumama
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17109384, 17113491 } }
-    },
-
-[ 898] = {  -- Feuerunke
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17334552, 17338598 } }
-    },
-
-[1758] = {  -- Tammuz
-        check = checks.mobkill,
-        reqs = { mobid = set{ 17195484 } }
-    },
+[   2] = { check = checks.checkTrials, reqs = { mobid = set{ 17563801 } } }, -- Nocuous Weapon
```

Comment:
Thank you for your feedback, Given the large number of trials I thought it would be more readable like that (1 line = 1 trial).
After that, if you have to change and put on several lines, that's not a problem. :)

Reply:
In lieu of being able to put this into the database because the format is a little flexible, I think I like the single-line format 👍